axiom/nr_daemon_spawn.c

```diff
@@ -144,7 +144,8 @@ nr_argv_t* nr_daemon_args_to_argv(const char* name,
 }
 
 #ifdef DO_GCOV
-extern void __gcov_flush(void);
+extern void __gcov_reset(void);
```

Comment:
I'm curious why this change?

Reply:
See 92eadcf8519e8c9b20bdb3c6bd8ce8029a3fc074.